TESTING_GUIDE.md

```diff
@@ -37,6 +47,10 @@ export OPENSEARCH_PASSWORD=<<your_password>>
 cd tests/default
 docker compose up -d
 ```
+In Windows env:
+``` 
+$Env:OPENSEARCH_PASSWORD="MyPassword!"
```

Comment:
```suggestion
$Env:OPENSEARCH_PASSWORD="<<your password>>"
```

To match the one above. Or replace <<your password>> with the default password we have `MyStrongPassword123!`.

Reply:
ok